release-notes/docs/3.10.0.md

```diff
@@ -0,0 +1,149 @@
+- [cht-core#6370](https://github.com/medic/cht-core/issues/6370): Replication "isSensitive" logic prevents "boss" from creating any report for users directly under them in hierarchy
+- [cht-core#6386](https://github.com/medic/cht-core/issues/6386): Error on report not cleared
+- [cht-core#6404](https://github.com/medic/cht-core/issues/6404): User name autofilled with "medic" 
+- [cht-core#6433](https://github.com/medic/cht-core/issues/6433): Infinite spinners when loading tabs as admin
+- [cht-core#6502](https://github.com/medic/cht-core/issues/6502): Guided tour erratic when user has no access to any tab
+- [cht-core#6519](https://github.com/medic/cht-core/issues/6519): Upgrading doesn't apply replication configuration which means `users-meta` is empty
+- [cht-core#6562](https://github.com/medic/cht-core/issues/6562): Muso production users have multiple task documents in non-terminal state with the same emission id
+- [cht-core#6567](https://github.com/medic/cht-core/issues/6567): Security issue medic-infrastructure#147
+- [cht-core#6568](https://github.com/medic/cht-core/issues/6568): Security issue medic-infrastructure#190
+- [cht-core#6575](https://github.com/medic/cht-core/issues/6575): Large grey banner shown above content on mobile
+- [cht-core#6578](https://github.com/medic/cht-core/issues/6578): Wrong error message shown when editing a user that replicates too many docs
+- [cht-core#6582](https://github.com/medic/cht-core/issues/6582): Scheduled messages with due date of `null` end up being processed over and over without being updated
+- [cht-core#6583](https://github.com/medic/cht-core/issues/6583): Users are forced to login one year after they last logged in
+- [horticulturalist#57](https://github.com/medic/horticulturalist/issues/57): Horti doesn't wam all views when deploying 3.9.x+
+- [medic-conf#290](https://github.com/medic/medic-conf/issues/290): Tests fail locally when Bash completion is not setup
+- [medic-conf#315](https://github.com/medic/medic-conf/issues/315): Error running csv-to-docs with null properties
+- [medic-conf#322](https://github.com/medic/medic-conf/issues/322): Missing optional appliesIf attribute on contact summary cards causes crash
+- [medic-conf#332](https://github.com/medic/medic-conf/issues/332): Uploading of multimedia is extremely limited in medic-conf
+
+### Technical issues
+
+- [cht-core#6369](https://github.com/medic/cht-core/issues/6369): Detect unhandled rejections in unit tests
+- [cht-core#6379](https://github.com/medic/cht-core/issues/6379): Figure out how to deploy RapidPro in production
+- [cht-core#6439](https://github.com/medic/cht-core/issues/6439): Auth service unit test is flaky
+- [cht-core#6463](https://github.com/medic/cht-core/issues/6463): Bump dependencies for 3.10
+- [medic-android#111](https://github.com/medic/medic-android/issues/111): Compliance with Play Store developer policies for PII collection disclosure
```

Comment:
This shouldn't be a technical issue...

Reply:
Uhm, ok... improvement then maybe? (you tagged it technical issue)

Reply:
The more I think about it, the more I think the technical  issue label is accurate. 

Existing users (that have already installed the app) should notice no difference (because they've already granted permissions when first running or upon installation?).

Reply:
> you tagged it technical issue

Hah. Well one of me is wrong. I'm not too bothered which. It's sort of an App Builder's feature? Anyway, just ignore me.